docs/src/modules/components/DemoEditor.tsx

```diff
@@ -16,7 +16,7 @@ const StyledMarkdownElement = styled(MarkdownElement)(({ theme }) => [
       overflow: 'auto',
       marginTop: -1,
       backgroundColor: 'hsl(210, 25%, 9%)', // a special, one-off, color tailored for the code blocks using MUI's branding theme blue palette as the starting point. It has a less saturaded color but still maintaining a bit of the blue tint.
-      border: 0,
+      border: '1px solid transparent',
```

Comment:
That's weird

Reply:
To fix the layout shift caused by border width (in light mode, no border but in dark mode the border is `1px`). This is noticeable in pages that have a lot of code block.

**Before (notice the text at the bottom of the clip)**:

https://github.com/user-attachments/assets/7a4b687b-467f-4a65-817d-7ccd70cc7e9e

**after (no longer shift)**:


https://github.com/user-attachments/assets/93d08e00-3679-4a01-9b46-f3dd2f5a4b37